deploy/manifests/dev/us-east-2/tenant/storetheindex/assigner/ingress.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: assigner
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
+    cert-manager.io/cluster-issuer: "letsencrypt"
+    nginx.ingress.kubernetes.io/enable-cors: "true"
+spec:
+  tls:
+    - hosts:
+        - assigner.dev.cid.contact
```

Comment:
For this URL to start receiving `PUT` announce requests we also need to adjust the [corresponding couldfront origin ](https://github.com/filecoin-project/storetheindex/blob/02e6d61ebe1575ee910d2537f94f121448dccf36/deploy/infrastructure/dev/us-east-2/cloudfront.tf#L23). Those changes need to be `plan`ed and `apply`ed manually once the assigner service is up as a separate PR. See runbook for steps on how to do this.

Reply:
OK